sap/cli/strust.py

```diff
@@ -217,14 +217,15 @@ def upload(connection, args):
     return 0
 
 
-@CommandGroup.argument('paths', type=str, nargs='+',
+@CommandGroup.argument('-p', '--paths', type=str, nargs='+',
                        help='a file path containing X.509 Base64 certificate')
 @CommandGroup.argument('-l', '--algorithm', type=str, help='R,S,G,H,X - or other if you need, of PSE file', default='R')
 @CommandGroup.argument('-k', '--key-length', type=int, default=2048, help='Of PSE file')
 @CommandGroup.argument('-d', '--dn', type=str, help='Distinguished Name of PSE file', default=None)
 @CommandGroup.argument('-s', '--storage', action='append', default=[],
                        choices=[CLIENT_ANONYMOUS, CLIENT_STANDART, CLIENT_STANDARD, SERVER_STANDARD, ])
 @CommandGroup.argument('-i', '--identity', action='append', default=[])
+@CommandGroup.argument('-c', '--certificate', type=str, help='X.509 Base64 certificate', default='')
```

Comment:
I am sorry, I didn't tell you before. Please, do not change the command line interface this way. Keep the positional parameter "paths" and add a code detecting "-" in the parameters which should cause the certificate is read from stdin.

Reply:
No problem. I have added a support for stdin and detection of '-' sign. By the way, thanks for your support with this.